packages/nouns-contracts/contracts/governance/fork/NounsDAOFork.sol

```diff
@@ -216,11 +219,20 @@ library NounsDAOFork {
 
     /**
      * @notice Returns the number of nouns in supply minus nouns owned by the DAO, i.e. held in the treasury or in an
-     * escrow after it has closed.
+     * escrow after it has closed, minus nouns hold via holding the nouns fungible token.
      * This is used when calculating proposal threshold, quorum, fork threshold & treasury split.
      */
     function adjustedTotalSupply(NounsDAOTypes.Storage storage ds) internal view returns (uint256) {
```

Comment:
@eladmallel something to consider and maybe check:
we're increasing the gas cost of this function.
by how much?
which functions call this and get their gas cost increased due to it?

Reply:
the change costs 2 cold storage reads (2100 gas x 2 = 4200) and an external call from the DAO to the fungible token.

`adjustedTotalSupply` is used in:

* submitting proposals
* fork functions like execute fork and join fork

this change adds less than 1% to the gas cost of proposing, therefore I think it's definitely worth it.